src/index.js

```diff
@@ -38,8 +38,19 @@ const renderTodoList = () => {
       }
     });
 
+    const deleteButton = document.createElement('button');
+    deleteButton.classList.add('delete-btn');
+    deleteButton.innerHTML = '<i class="ri-delete-bin-line"></i>';
+
+    deleteButton.addEventListener('click', () => {
+      tasks.splice(index, 1);
+      updateLocalStorage();
+      renderTodoList();
+    });
```

Comment:
- Good job done so far, but according to the project requirements, you are required to have a new  JS file that contains the new functionalities. Doing this will help make your project more modular and easier to maintain. Please fix this 🙏 
- Also according to the project requirements, by default, new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5).
Your project:
<img width="460" alt="wrawra" src="https://github.com/aradradev/to-do-list-day1/assets/73607512/06721b7b-7fa1-420e-806e-16556eb7682a">
- After implementing the above suggestion, according to the project requirements, you are to ensure that `deleting a task should update all remaining items' indexes, so they represent the current list order and are unique`. Please fix this 😄

Reply:
- Yes everything is working fine with respect to adding and removing tasks, but your task object is missing the `index` key when adding a new task. The value of this `index` key should be set to the value of the new array length as mentioned in the project requirements. I hope You understand better now.